msal/build.gradle

```diff
@@ -21,10 +21,10 @@ android {
 
     buildTypes {
         debug {
-            testCoverageEnabled true
+            testCoverageEnabled false
         }
         release {
-            testCoverageEnabled true
+            testCoverageEnabled false
```

Comment:
Temporarily disabled testCoverage - was throwing `java.lang.NoClassDefFoundError: Failed resolution of: Lorg/jacoco/agent/rt/internal_773e439/Offline`